extensions/src/ACRE2Shared/TextMessage.cpp

```diff
@@ -22,15 +22,19 @@ acre::Result CTextMessage::parse(char *const value, const size_t len) {
         return acre::Result::ok;
     }
 
-    // Check to make sure the entire chunk of data is a NULL terminated ascii string
-    for (size_t x = 0 ; x < len; x++) {
-        if (!__isascii(value[x]) && value[x] != 0x00) {
-            this->m_IsValid = false;
-            LOG("INVALID PACKET DETECTED l:%d", len);
-            return acre::Result::error;
-        }
-        if (value[x] == 0x00) {   // null terminate, bail
-            break;
+    // Don't check for ascii only characters for setTs3ChannelDetails procedure
+    std::string text(value);
+    if (text.find("setTs3ChannelDetails") == std::string::npos) {
+        // Check to make sure the entire chunk of data is a NULL terminated ascii string
+        for (size_t x = 0; x < len; x++) {
```

Comment:
```suggestion
        for (size_t x = 0U; x < len; x++) {
```

Reply:
This line wasn't edited by me but have made the change anyway